types/mirrorx/tslint.json

```diff
@@ -0,0 +1,25 @@
+{
+    "extends": "dtslint/dt.json",
```

Comment:
Means that for new packages, this file should just `"extends": "dtslint/dt.json"`, no other rules.